ons_alpha/settings/base.py

```diff
@@ -52,6 +52,9 @@
 if "CSRF_TRUSTED_ORIGINS" in env:
     CSRF_TRUSTED_ORIGINS = env["CSRF_TRUSTED_ORIGINS"].split(",")
 
+WAGTAILADMIN_BASE_URL = env.get("WAGTAILADMIN_BASE_URL", "http://localhost:8000")
```

Comment:
already defined in https://github.com/ONSdigital/dis-wagtail-alpha/blob/main/ons_alpha/settings/base.py#L691-L695

if you need to make changes to any of the already defined settings locally, please add them to `ons_alpha/settings/local.py`